cvat/apps/engine/views.py

```diff
@@ -616,10 +614,32 @@ def data(self, request, pk):
                     status=status.HTTP_400_BAD_REQUEST)
             serializer = DataSerializer(data=request.data)
             serializer.is_valid(raise_exception=True)
+            data = {k: v for k, v in serializer.validated_data.items()}
+
+            tmp_path = os.path.join('./data/tmp', pk)
+            os.makedirs(tmp_path, exist_ok=True)
+            if data.get('use_chunk_upload'): # write chunk
+                with open(os.path.join(tmp_path, data.get('chunk_file_name')), 'ab+') as destination:
```

Comment:
If ``chunk_file_name`` exists, ``use_chunk_upload`` also always true, right? 
I think ``use_chunk_upload`` is extra here.

Reply:
Removed